tests/lib/screenshot-testing/support/page-renderer.js

```diff
@@ -258,9 +271,12 @@ PageRenderer.prototype.capture = function (outputPath, callback, selector) {
     var self = this,
         timeout = setTimeout(function () {
             var timeoutDetails = "";
-            timeoutDetails += "Page not ready: " + self._isLoading + "\n";
+            timeoutDetails += "Page is loading: " + self._isLoading + "\n";
+            timeoutDetails += "Initializing: " + self._isInitializing + "\n";
+            timeoutDetails += "Navigation requested: " + self._isNavigationRequested + "\n";
             timeoutDetails += "Pending AJAX request count: " + self._getAjaxRequestCount() + "\n";
             timeoutDetails += "Loading images count: " + self._getImageLoadingCount() + "\n";
+            timeoutDetails += "Remaining resources: " + JSON.stringify(self._resourcesRequested) + "\n";
```

Comment:
I do not really think it would be much clearer and we would lose the information whether one resource is pending multiple times. I can change if you prefer. I do not really care since it should not be shown often and if so, it will be helpful either way